src/Iot.Device.Bindings.SkiaSharpAdapter/Directory.Build.props

```diff
@@ -0,0 +1,13 @@
+<Project>
+  <!-- Packaging related properties -->
+  <PropertyGroup>
+    <MajorVersion>2</MajorVersion>
+    <MinorVersion>3</MinorVersion>
+    <Description>This package contains the BitmapImage adapter using SkiaSharp. This library is an optional package to provide imaging functionality for Iot.Device.Bidnings.dll</Description>
+    <PackageTags>.NET Core IoT Device Bindings SkiaSharp Adapter</PackageTags>
+    <EnablePackageValidation>false</EnablePackageValidation>
+    <PackageValidationBaselineVersion>2.2.0</PackageValidationBaselineVersion>
```

Comment:
I'm fine if you want to keep this as is since PackageValidation is disabled, but it might be better to just introduce this property when we actually enable PackageValidation.